compiler+runtime/src/cpp/jank/util/cli.cpp

```diff
@@ -71,11 +71,20 @@ namespace jank::util::cli
     /* C++ REPL subcommand. */
     auto &cli_cpp_repl(*cli.add_subcommand("cpp-repl", "Start up a terminal C++ REPL."));
 
-    /* Run subcommand. */
+    /* run-main subcommand. */
     auto &cli_run_main(*cli.add_subcommand("run-main", "Load and execute -main."));
     cli_run_main.fallthrough();
     cli_run_main.add_option("module", opts.target_module, "The entrypoint module.")->required();
 
+    /* compile-main subcommand. */
+    auto &cli_compile_main(
+      *cli.add_subcommand("compile-main",
+                          "Load and compile project with entrypoint module containing -main."));
```

Comment:
What do you think about:

1. Renaming `compile-main` to just `compile`
2. Renaming the existing `compile` to `compile-module`

Reply:
I was thinking the same. I will make that change.